bm-persona/Libraries/LibraryDetailViewController.swift

```diff
@@ -49,156 +40,101 @@ class LibraryDetailViewController: SearchDrawerViewController {
-//        overviewCard.rightAnchor.constraint(equalTo: scrollView.layoutMarginsGuide.rightAnchor).isActive = true
-//        overviewCard.heightAnchor.constraint(equalToConstant: 200).isActive = true
-//    }
-//
-    
     func setUpBookButton() {
-        bookButton.isUserInteractionEnabled = true
-        bookButton.layoutMargins = kCardPadding
-        scrollView.addSubview(bookButton)
-        bookButton.backgroundColor =  Color.eventAcademic
-        bookButton.layer.cornerRadius = 10
-        bookButton.layer.shadowRadius = 5
-        bookButton.layer.shadowOpacity = 0.25
-        bookButton.layer.shadowOffset = .zero
-        bookButton.layer.shadowColor = UIColor.black.cgColor
-        bookButton.layer.shadowPath = UIBezierPath(rect: bookButton.layer.bounds.insetBy(dx: 4, dy: 4)).cgPath
-        bookButton.translatesAutoresizingMaskIntoConstraints = false
-        bookButton.topAnchor.constraint(equalTo: hoursCard.layoutMarginsGuide.bottomAnchor, constant: kViewMargin).isActive = true
-        bookButton.leftAnchor.constraint(equalTo: scrollView.layoutMarginsGuide.leftAnchor).isActive = true
-        bookButton.rightAnchor.constraint(equalTo: scrollView.layoutMarginsGuide.rightAnchor).isActive = true
+        scrollContent.addSubview(bookButton)
+        bookButton.topAnchor.constraint(equalTo: openTimesCard?.bottomAnchor ?? overviewCard.bottomAnchor, constant: kViewMargin).isActive = true
+        bookButton.leftAnchor.constraint(equalTo: scrollContent.leftAnchor).isActive = true
+        bookButton.rightAnchor.constraint(equalTo: scrollContent.rightAnchor).isActive = true
+        bookButton.bottomAnchor.constraint(equalTo: scrollContent.bottomAnchor).isActive = true
         bookButton.heightAnchor.constraint(equalToConstant: 50).isActive = true
-        bookButton.setTitle("Book a Study Room >", for: .normal)
-        bookButton.titleLabel!.font = Font.semibold(14)
-        bookButton.titleLabel!.textColor = .white
-        
     }
     
     func setUpScrollView() {
-        scrollView = UIScrollView(frame: .zero)
-        scrollView.contentInsetAdjustmentBehavior = .never
-        scrollView.layoutMargins = UIEdgeInsets(top: 0, left: 10, bottom: 0, right: 10)
-        scrollView.alwaysBounceVertical = true
-        view.addSubview(scrollView)
-        scrollView.translatesAutoresizingMaskIntoConstraints = false
-        scrollView.topAnchor.constraint(equalTo: barView.topAnchor, constant: kViewMargin).isActive = true
-        scrollView.leftAnchor.constraint(equalTo: view.layoutMarginsGuide.leftAnchor).isActive = true
-        scrollView.rightAnchor.constraint(equalTo: view.layoutMarginsGuide.rightAnchor).isActive = true
-        scrollView.bottomAnchor.constraint(equalTo: view.layoutMarginsGuide.bottomAnchor).isActive = true
+        view.addSubview(scrollContainer)
+        scrollContainer.topAnchor.constraint(equalTo: barView.topAnchor, constant: kViewMargin).isActive = true
```

Comment:
Why did you choose to compare both top anchors instead of the vertical distance?

Reply:
was a mistake